romsel_aktheme/arm9/source/windows/mainlist.cpp

```diff
@@ -239,11 +283,11 @@ bool MainList::enterDir(const std::string &dirName)
 
         if (sdFound())
         {
-            addDirEntry(LANG("mainlist", ((ms().showMicroSd) ? "microSD Card" : "SD Card")), "", (ms().showDirectories ? SD_ROOT : ms().romfolder[0]), "usd", microsd_banner_bin);
+            addDirEntry(LANG("mainlist", ((ms().showMicroSd) ? "microSD Card" : "SD Card")), "", (ms().showDirectories ? SD_ROOT : ms().getPrimaryRomFolder()), "usd", microsd_banner_bin);
         }
         if (flashcardFound())
         {
-            addDirEntry(LANG("mainlist", ((sys().isRegularDS()) ? "microSD Card" : "SLOT-1 microSD Card")), "", (ms().showDirectories ? S1SD_ROOT : ms().romfolder[1]), "usd", microsd_banner_bin);
+            addDirEntry(LANG("mainlist", ((sys().isRegularDS()) ? "microSD Card" : "SLOT-1 microSD Card")), "", (ms().showDirectories ? S1SD_ROOT : ms().getSecondaryRomFolder()), "usd", microsd_banner_bin);
         }
         addDirEntry("GBARunner2", "", SPATH_GBARUNNER, "gbarunner", gbarom_banner_bin);
```

Comment:
We don't need this anymore.

Reply:
What do you mean

Reply:
GBA ROMs can now be launched from the menu itself. We don't need a dedicated GBARunner2 button. Otherwise, we should have a dedicated nesDS button, a SNEmulDS button, etc.